bundles/org.openhab.binding.pixometer/src/main/resources/ESH-INF/thing/gasmeter.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="pixometer"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="gasmeter">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="apiservice" />
+		</supported-bridge-type-refs>
+		<label>Gasmeter</label>
+		<description>Provides data for one specific meter.</description>
+		<channels>
+			<channel id="last_reading_value" typeId="last_reading_value" />
+			<channel id="last_reading_date" typeId="last_reading_date" />
+			<channel id="last_refresh_date" typeId="last_refresh_date" />
+		</channels>
+		<config-description>
+			<parameter name="resource_id" type="text" required="true">
+				<label>Ressource ID</label>
+				<description>The ID which represents the current meter. You can find it in the pixometer browser app while editing a specific meter. It should look like this: "https://pixometer.io/portal/#/meters/XXXXX/edit"</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<channel-type id="last_reading_value">
```

Comment:
You must define a channel type only once in the binding.
So better define different types for the different meters, e.g. `last_reading_energy`and `last_reading_gas` etc.

Reply:
How would i handle this in the thing handler then best?
Would i have to separate the current type of thiong and the update the channel with some decision logic?

Reply:
In the code you don't need to change anything. The channel `id` here can be the same for all. Just the `typeId` should be different. to differentiate you could append type, like `last_reading_value_type`. Speaking of last reading value. I guess it would make more sense to use a label that actual states what is read. Like `Gas Volume`.

Reply:
🤦‍♂ 

Sorry, i had a look at the xml files again and it seems i had a long dayyesterday...
The change is indeed pretty easy after taking a fresh look.